<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>533</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Programme</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="keyLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="keyTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Message or File Path</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="dataTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="algorithmLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QComboBox" name="algorithmComboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="currentText">
     <string notr="true">AES</string>
    </property>
    <item>
     <property name="text">
      <string>AES</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blowfish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DES</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XTEA</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="modeComboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true">CBC</string>
    </property>
    <item>
     <property name="text">
      <string>CBC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CFB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ECB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OFB</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="resultTextBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateKeyButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Key</string>
    </property>
   </widget>
   <widget class="QComboBox" name="paddingComboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true">No Padding</string>
    </property>
    <item>
     <property name="text">
      <string>No Padding</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zero Padding</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO 10126</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PKCS5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="paddingLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Padding</string>
    </property>
   </widget>
   <widget class="QComboBox" name="keySizeComboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>128</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="keySizeLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Key Size in Bits</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copyKeyButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resultCopyButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isFileCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Are you decrypting a file?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseInputFileButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resultFilePickButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose file destination</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="IVTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="IVLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Programme</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter IV (optional)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="10" margin="10"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
